cli/tests/unit/webcrypto_test.ts

```diff
@@ -1,5 +1,62 @@
 import { assert, assertEquals, assertRejects, unitTest } from "./test_util.ts";
 
+unitTest(async function testDigest() {
+  const subtle = window.crypto.subtle;
+  assert(subtle);
+
+  {
+    const result = await subtle.digest({ name: "MD-5" }, new Uint8Array([42]));
```

Comment:
Is "MD-5" name compatible with CFW? I couldn't find an example in their docs.

Reply:
IIRC Cloudflare MD5 algorthim is named "MD5" (without the `-`)

Reply:
Good catch, updated.